engine/access/rpc/backend/retry.go

```diff
@@ -95,7 +94,7 @@ func (r *Retry) prune(height uint64) {
 	}
 }
 
-func (r *Retry) retryTxsAtHeight(heightToRetry uint64) {
+func (r *Retry) retryTxsAtHeight(heightToRetry uint64) error {
```

Comment:
Since this function returns errors we need to write a godoc which described possible outcomes